packages/clay-form/src/MultiSelect.tsx

```diff
@@ -0,0 +1,319 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import ClayAutocomplete from '@clayui/autocomplete';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import React, {useLayoutEffect, useRef, useState} from 'react';
+
+const COMMA = 188;
+
+const KEYS = [COMMA];
+
+const BACKSPACE_KEY = 8;
+
+export interface IProps extends React.HTMLAttributes<HTMLInputElement> {
+	/**
+	 * List of key codes that will trigger onItemsChange(inputValue).
+	 */
+	deliminatorKeys?: number[];
+
+	/**
+	 * Message to display if validation fails.
+	 */
+	errorMessage?: string;
+
+	/**
+	 * Ref added to container element
+	 */
+	forwardRef: React.RefObject<HTMLDivElement>;
+
+	/**
+	 * Message to display which helps users interact with input
+	 */
+	helpText?: string;
+
+	/**
+	 * Value of input
+	 */
+	inputValue?: string;
+
+	/**
+	 * Values that display as label items
+	 */
+	items: string[];
+
+	/**
+	 * Description of the input
+	 */
+	label?: React.ReactText;
+
+	/**
+	 * Callback for when items are added or removed
+	 */
+	onItemsChange: (val: string[]) => void;
+
+	/**
+	 * Callback for when the input value changes
+	 */
+	onInputChange?: (val: string) => void;
+
+	/**
+	 * List of pre-populated items that will show up in a dropdown menu
+	 */
+	sourceItems?: string[];
+
+	/**
+	 * Path to spritemap for clay icons
+	 */
+	spritemap?: string;
+
+	/**
+	 * Flag to indicate if validation should occur when input is blured
+	 */
+	validateOnBlur?: boolean;
+
+	/**
+	 * Callback for validating item. Returning `true` means the value is valid.
+	 */
+	validationFn?: (val?: string) => boolean;
```

Comment:
None of the other callback names have an `Fn` suffix, so why would this one? The types tell us what it is anyway. Could it be `validator` or `onValidate` instead?